### DISCUSSION TITLE
Upload file Failure  in PUMA 6.4.2

### DISCUSSION THREAD

**Discussion by shashankposhmark** [puma/puma#3448] [Q&A] [answered]
After upgrading PUMA from 5.6.7 to 6.4.2 , in my android applications i am unable to upload videos using mobile data (low speed and low stability).
while using puma 5.6.7, I didn’t face any issue. 
want to know is there any specific configuration that will fix this issue?

**Comment by dentarg** [comment]
Unable how? Please provide as much details as you can. If you can produce instructions, code, commands how to reproduce the problem, there will be much greater chance getting it resolved.

**Comment by dentarg** [comment]
Can you try Puma [5.6.8](https://github.com/puma/puma/blob/1ef63c25ec41aa30b4ebfe99f8e5ffc966bbe00b/History.md#568--2024-01-08)?

**Comment by shashankposhmark** [comment]
I’m encountering the following error: <Puma::HttpParserError: maximum size of chunk header exceeded>. This issue occurs in both Puma version 5.6.8 and 6.4.2. I noticed a similar problem was addressed in a patch released three months ago, but these changes do not appear to be reflected in version 6.4.2.
https://github.com/puma/puma/issues/3337

**Reply by dentarg** [reply]
They will be in the next release

**Reply by shashankposhmark** [reply]
any tentative date for next release?

**Comment by shashankposhmark** [comment]
Is it possible for me to configure the value of MAX_CHUNK_HEADER_SIZE in Puma?

**Comment by shashankposhmark** [comment]
I can see these fix in master branch 
https://github.com/puma/puma/blob/master/lib/puma/client.rb#L653

**Comment by shashankposhmark** [comment]
why these changes are not in v6.4.3 
https://github.com/puma/puma/commit/ebb40b620ffe5b28be401d9f697b857bc94d5731
I can see these fix in master branch
https://github.com/puma/puma/blob/master/lib/puma/client.rb#L653

**Reply by dentarg** [reply, marked as answer]
v6.4.3 was security fix only
